api/flatbuffers/nanoapi.fbs

```diff
@@ -93,6 +93,7 @@ enum BlockSubType: byte {
 /** Block union */
 union Block {
 	BlockState,
+	BlockState2,
```

Comment:
Strictly speaking, this should be appended at the end of the union to ensure backwards compatibility, but I don't think fb is used yet, so maybe it's fine for now?

Reply:
Good point, could cause some confusion in the future if we add another state block version and just append it in the middle so will move to end.